chromium/janitor.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "Chromium",
+  "description": "The Chromium browser.",
```

Comment:
I think they're all dev environments, so I don't think that's worth mentioning. I think the description field should contain a description of the project, and maybe why you should contribute to it.